item-seller/readme.md

```diff
@@ -1,4 +1,15 @@
-This guide will walk you through the process of building contracts, deploying them into an existing world running in Docker, and testing their functionality by executing scripts.
+## Introduction
+This guide will walk you through the process of building contracts for the item seller, deploying them into an existing world running in Docker, and testing their functionality by executing scripts. The contracts will be used with a SSU (Smart Storage Unit) to act as a item seller where players can purchase items, for example lenses for ERC20 Tokens.
+
+### User Process
+#### Step 0: User buys an item
+The item seller will recieve the ERC20 token and authorize the transaction if the player has paid enough.
```

Comment:
This section can be refactored a bit like 

```
### user flow 

The item seller is a Smart Storage Unit(SSU) which can accept ERC20 tokens and transfer items to the players. 

The SSU (the item seller in this case) maintains two types of inventories:

Inventory: This is the storage space that belongs to the SSU owner.
Ephemeral Inventory: A temporary storage area for players interacting with the SSU. Players can deposit and withdraw items from the SSU via the ephemeral inventory.

When an ERC20 token is transferred by the player, the item seller SSU transfers items from the owner's inventory to the player’s ephemeral inventory. Players can then withdraw these items to their ship's hangar.
```

Reply:
Have refactored.